neptune/alpha/project.py

```diff
@@ -0,0 +1,58 @@
+#
+# Copyright (c) 2020, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import uuid
+from typing import Union, Optional, Iterable
+
+from neptune.alpha.internal.backends.hosted_neptune_backend import HostedNeptuneBackend
+from neptune.alpha.internal.utils import verify_type, verify_collection_type
+from neptune.alpha.leaderboard import Leaderboard
+
+
+class Project:
+
+    def __init__(self,
+                 _uuid: uuid.UUID,
+                 backend: HostedNeptuneBackend):
+        self._uuid = _uuid
+        self._backend = backend
+
+    def get_table(self,
+                  _id: Optional[Union[str, Iterable[str]]] = None,
+                  state: Optional[Union[str, Iterable[str]]] = None,
+                  owner: Optional[Union[str, Iterable[str]]] = None,
+                  tag: Optional[Union[str, Iterable[str]]] = None,
+                  min_running_time: Optional[int] = None
+                  ) -> Leaderboard:
+        _id = self._as_list("_id", _id)
```

Comment:
It's user interface so let's call it `id` and ignore warning. Do the same in global function.